inst/include/common/information.hpp

```diff
@@ -43,6 +43,8 @@ class Information {
       random_effects_parameters; /**< list of all random effects parameters >*/
   std::vector<Type*>
       fixed_effects_parameters; /**< list of all fixed effects parameters >*/
+  std::vector<std::string>
+      parameter_names; /**< list of all parameter names estimated in the model */
```

Comment:
Question: I noticed that fims::Vector is being used in data_object.hpp. Do we need to refactor std::vector to fims::Vector in other parts of the codebase?

Reply:
We will be going through the codebase and refactoring std::vector to fims::Vector where needed before our final sprint in September.